vizro-core/tests/component_library/test_component_library.py

```diff
@@ -0,0 +1,93 @@
+# ruff: noqa: F403, F405
```

Comment:
What are these ignores?

If justified then I'm happy to add more to `[tool.ruff.lint.per-file-ignores]` that works across all `tests` files, but so far we haven't ignored there's so would be good to understand why we need to now.